### PR TITLE
Rename appliance to managed application

### PR DESCRIPTION
**Pull request by vivsriaus**
Rename appliance to managed application, and update api version

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by sarangan12** [comment]
@vivsriaus You have marked this PR do not merge. Is that still valid? Do you want this PR merge?

**Comment by sarangan12** [comment]
@vivsriaus You have marked this PR do not merge. Is that still valid? Do you want this PR merge?

**Comment by vivsriaus** [comment]
@sarangan12 we're still waiting to finalize the api on our service end. I should be able to get back on this by early next week. Please keep the PR active for now.

**Comment by vivsriaus** [comment]
@ravbhatnagar fyi

**Comment by sarangan12** [comment]
@vivsriaus Please take a look at https://travis-ci.org/Azure/azure-rest-api-specs/builds/274775514?utm_source=github_status&utm_medium=notification 

Under allowed failures, specifically look at linter, model and semantic validations. The errors must be fixed for the approval of this PR. 

Please fix the errors and update the PR

**Comment by vivsriaus** [comment]
@sarangan12 Not sure why this error shows up. I don't see duplicate paths in my file.

{ code: 'SEMANTIC_VALIDATION_ERROR',
  id: 'OAV119',
  message: 'The spec /home/travis/build/Azure/azure-rest-api-specs/specification/resources/resource-manager/Microsoft.Solutions/2017-09-01/managedapplications.json has semantic validation errors.',
  innerErrors: 
   [ { code: 'EQUIVALENT_PATH',
       message: 'Equivalent path already exists: /{applicationDefinitionId}',
       path: [ 'paths', '/{applicationDefinitionId}' ] } ] }

**Comment by vivsriaus** [comment]
There are no microsoft.solutions errors in model, and in linter, all the errors are for the older file, not for the new one. Can you please take a look? @sarangan12

File: specification/resources/resource-manager/Microsoft.Solutions/2017-09-01/examples/createOrUpdateApplication.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
```
Comment (gandhiniraj) [inline specification/resources/resource-manager/Microsoft.Solutions/2017-09-01/examples/createOrUpdateApplication.json]: why not full name? subscriptionId?

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**